package.json

```diff
@@ -8,11 +8,16 @@
     "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
     "build": "rm -rf build && babel -d ./build ./src -s",
     "start": "npm run build && node ./build/index.js",
-    "pretest": "cross-env NODE_ENV=test npm run down && NODE_ENV=test npm run migrate  && NODE_ENV=test npm run seed",
-    "test": "cross-env NODE_ENV=test nyc --reporter=text mocha  --require @babel/register --require @babel/polyfill  ./src/tests/*.js --timeout 30000 --exit",
+    "mocha-test": "nyc --reporter=text mocha  --require @babel/register --require @babel/polyfill  ./src/tests/*.js --timeout 30000 --exit",
+    "test": "cross-env NODE_ENV=test npm-run-all db:ready mocha-test",
     "down": "cross-env NODE_ENV=test node_modules/.bin/sequelize db:migrate:undo",
+    "db:ready": "npm-run-all down migrate seed",
     "migrate": "cross-env NODE_ENV=test node_modules/.bin/sequelize db:migrate",
-    "seed": "node_modules/.bin/sequelize db:seed:all"
+    "seed": "node_modules/.bin/sequelize db:seed:all",
+    "unseed:all": "node_modules/.bin/sequelize db:seed:undo:all",
+    "undo:seed": "node_modules/.bin/sequelize db:seed:undo:all",
+    "dropdb": "node_modules/.bin/sequelize db:drop",
```

Comment:
It seems that is not being used. So remove those two script
`"dropdb": "node_modules/.bin/sequelize db:drop",`
`"createdb": "node_modules/.bin/sequelize db:create authors_haven_test"`

Reply:
Right! Done!